aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -257,17 +263,35 @@ When using the out-of-process hosting model, the app might not shut down immedia
 ::: moniker-end
 
 ## Start-up error page
-
 ::: moniker range=">= aspnetcore-2.2"
 
-*Only applies to out-of-process hosting.*
+Both inprocess and outofprocess have custom error pages when they fail to start the application.
+
+If the ASP.NET Core module fails to find either the inprocess or out of process request handler, a *500.0 In-Process/Out-Of-Process Handler Load Failure* status code page appears.
```

Comment:
* "ASP.NET Core module" :point_right: "ASP.NET Core Module" (capitalize)
* "in-process or out-of-process" (hyphens)
* Use a dash between the status code and message ... `*500.0 - In-Process/Out-Of-Process Handler Load Failure*`

Cut-'n-paste ...

> If the ASP.NET Core Module fails to find either the in-process or out-of-process request handler, a \*500.0 - In-Process/Out-Of-Process Handler Load Failure\* status code page appears.